.github/workflows/links.yml

```
@@ -1,45 +1,37 @@
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v2
 
-      - name: Restore lychee cache
-        id: restore-cache
-        uses: actions/cache/restore@v4
+      - name: Cache lychee
+        id: cache-lychee
+        uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)